src/core/types/saved_objects.ts

```diff
@@ -115,7 +115,7 @@ export interface SavedObject<T = unknown> {
    */
   originId?: string;
   /** Workspaces that this saved object exists in. */
-  workspaces?: string[];
+  workspaces?: string[] | null;
```

Comment:
This is SavedObject interface, the `workspaces` field cannot be `null` on workspace object, am I right?

Reply:
No, workspaces cannot be null on workspace object. But workspaces field on saved object interface can be `undefined` and I want to unify the workspaces type to one source. Let me try to find if there is any other place that we can put the null type to.

Reply:
In the `SavedObjectsBaseOptions`?

Reply:
Exactly, done for that.